account/address.go

```diff
@@ -0,0 +1,84 @@
+package account
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hyperledger/burrow/word"
+	"github.com/tmthrgd/go-hex"
+	"golang.org/x/crypto/ripemd160"
+)
+
+type Address word.Word160
+
+var ZeroAddress = Address{}
+
+func AddressFromBytes(addr []byte) (address Address, err error) {
+	if len(addr) != word.Word160Length {
+		err = fmt.Errorf("slice passed as address '%X' has %d bytes but should "+
+			"have %d bytes", addr, len(addr), word.Word160Length)
+		// It is caller's responsibility to check for errors. If they ignore the error we'll assume they want the
+		// best-effort mapping of the bytes passed to an  address so we don't return here
+	}
+	copy(address[:], addr)
+	return address, nil
```

Comment:
It looks like this should be a naked return, otherwise you'll throw away the error you've potentially created above.

Reply:
Good spot, thanks